### ISSUE TITLE
Speech Recogonition cannot be reached in system tray with NVDA with WSR active, when machine returns back from sleep 

### ISSUE THREAD

**Issue by sumandogra** [dictationbridge/dictationbridge-nvda#22]
Windows 10
Suggestion:
WSR with NVDA and dictation bridge, No dragon
After leaving the machine unused for some period of time and getting it back  after it sleeps, it  is difficult to reach out to the overflow notification area with NVDA sometimes, and one has to struggle to find the speech recognition command in the system tray with NVDA.   
Problems faced:
1.	overflow notification area cannot be reached easily with NVDA.
2.	If one reaches  it sometimes, only one menu item-NVDA is announced and one cannot reach to the speech recognition.  One has to close and restart NVDA in order to reach out to the speech recognition menu item. 
3.	As there are no help messages available in NVDA one may not know that one has to press ENTER on speech recognition menu item  to get the context menu.

Steps:
1.	 With NVDA running and dictation bridge add-on enabled, start Windows speech recognition on Windows.
2.	Get the start menu and type Windows speech recognition.
3.	Go to the system tray and simply search for the speech recognition.
4.	Leave the system unused for some time till it sleeps or make the machine sleep.
5.	Again sign in.
6.	Now restart with windows speech recogonition.
7.	Go to system tray with TAB.
8.	It becomes difficult to find the speech recognition command.

**Comment by derekriemer** [comment]
This is a duplicate of the other issue you filed, and I am also unable to reproduce it. Let's move discussion to #21
